src/components/data-processing/ConfigureEmbedding/CalculationConfig.jsx

```diff
@@ -81,115 +101,116 @@ const CalculationConfig = (props) => {
+    setChanges({ ...newChanges });
+  };
+  const setDistanceMetric = (value) => {
+    newChanges.embeddingSettings.methodSettings.umap.distanceMetric = parseFloat(value);
+    setChanges({ ...newChanges });
+  };
+  if (!changesOutstanding && !_.isEqual(changes, initialValues)) {
+    setChangesOutstanding(true);
+  }
+  if (changesOutstanding && _.isEqual(changes, initialValues)) {
+    setChangesOutstanding(false);
+  }
+  const renderUMAPSettings = () => {
+    const { umap } = changes.embeddingSettings.methodSettings;
+    return (
+      <>
+        <Form.Item>
+          <Text strong>Settings for UMAP:</Text>
+        </Form.Item>
+        <Form.Item label={(
+          <span>
+            Minimum distance&nbsp;
+            <Tooltip title={MIN_DIST_TEXT}>
+              <QuestionCircleOutlined />
+            </Tooltip>
+          </span>
+        )}
         >
-          <Option value='euclidean'>Euclidean</Option>
-          <Option value='cosine' disabled>Cosine</Option>
-        </Select>
-      </Form.Item>
-    </>
-  );
-
-  const renderTSNESettings = () => (
-    <>
-      <Form.Item>
-        <Text strong>Settings for t-SNE:</Text>
-      </Form.Item>
-      <Form.Item label='Perplexity:'>
-        <InputNumber
-          value={tsneSettings.perplexity}
-          min={5}
-          onInput={(e) => updateSettings(
-            {
-              embeddingSettings:
-                { methodSettings: { tsne: { perplexity: parseFloat(e.target.value) } } },
-            },
-          )}
-          onStep={(value) => updateSettings(
-            {
-              embeddingSettings:
-                { methodSettings: { tsne: { perplexity: parseFloat(value) } } },
-            },
-          )}
-        />
-      </Form.Item>
-      <Form.Item label='Learning rate:'>
-        <InputNumber
-          value={tsneSettings.learningRate}
-          min={10}
-          max={1000}
-          step={10}
-          onInput={(e) => updateSettings(
-            {
-              embeddingSettings:
-                { methodSettings: { tsne: { learningRate: parseFloat(e.target.value) } } },
-            },
-          )}
-          onStep={(value) => updateSettings(
-            {
-              embeddingSettings:
-                { methodSettings: { tsne: { learningRate: parseFloat(value) } } },
-            },
-          )}
+          <InputNumber
+            value={umap.minimumDistance}
+            min={0}
+            step={0.1}
+            onChange={(value) => setMinimumDistance(value)}
+            onStep={(value) => setMinimumDistance(value)}
+            onPressEnter={(e) => e.preventDefault()}
 
-        />
-      </Form.Item>
-    </>
-  );
+            onBlur={(e) => setMinimumDistance(e.target.value)}
+          />
+        </Form.Item>
+        <Form.Item label='Distance metric:'>
+          <Tooltip title='Cosine metric is going to be supported on a future version of the platform.'>
```

Comment:
this is the place that needs fixing according to my previous comment with the screenshot